More2Life/ImagePageViewController.swift

```diff
@@ -51,7 +51,7 @@ extension ImagePageViewController: UIPageViewControllerDataSource {
     // MARK: - Page Indicator
     
     func presentationCount(for pageViewController: UIPageViewController) -> Int {
-        return imageViewControllers.count
+        return imageViewControllers.count > 1 ? 0 : -1
```

Comment:
What is the idea behind this?

Reply:
this removes the page indicator if the page view has only one page. no need for the dot if it's by itself.

Reply:
Isn't this correct logic?

```
imageViewControllers.count > 1 ? imageViewControllers.count : -1
```

Reply:
duh